numpy/lib/_nanfunctions_impl.py

```diff
@@ -1646,14 +1646,20 @@ def _nanquantile_ureduce_func(
         wgt = None if weights is None else weights.ravel()
         result = _nanquantile_1d(part, q, overwrite_input, method, weights=wgt)
     else:
+        #GH-25731
```

Comment:
I would replace the gh-xxx by a short comment to ease reading the code - or remove the comment entirely.